### ISSUE TITLE
Configure App controllers as a collection instead of a String

### ISSUE THREAD

**Issue by ashmaroli** [ohler55/wabur#112]
Currently, Controller(s) is configured as a `String` matching the `WAB::Controller` subclass
```ruby
{
  'controller' => 'WAB::OpenController',
}
```
instead it'd be better if the Configuration Hash mapped the Controllers as a collection:
```ruby
{
  'controller' => {
    nil     => 'WAB::OpenController',
    'entry' => 'EntryController'
  }
}
```
or
```ruby
{
  'controller' => [
    { type => nil, 'class' => 'WAB::OpenController' },
    { type => 'entry', 'class' => 'EntryController' }
  ]
}
```

**Comment by ashmaroli** [comment]
Some alternative measures:
  - rename to **`config["controllers"]`** which will be an **Array of controller subclasses**. Then have WAB programmatically register the controller to the Shell instance based on its name. i.e. : 
    - The default controller is `WAB::Controller`. it is registered with type: `nil` or `WAB` or `Default`
    - `ArticleController` automatically registers with type: `Article`
    - `CommentsController` auto-registers with type: `Comments`
    - i.e. alphabet chars preceding `Controller` becomes the type.
  - compute an array of all subclasses of `WAB::OpenController` `require`d by an App and register all of them automatically with the shell instance. ( more error-prone ? )

**Comment by ohler55** [comment]
Addressed in PR #115
